gui-apps/waybar/metadata.xml

```diff
@@ -15,10 +15,15 @@
 	</maintainer>
 	<use>
 		<flag name="experimental">Enable experimental features</flag>
```

Comment:
Such as? :) are there any examples that this holds.

Reply:
I added a short example to clarify